### ISSUE TITLE
DS.Store#unloadRecord snippet highlighting broken

### ISSUE THREAD

**Issue by Turbo87** [ember-learn/ember-api-docs#464]
The syntax highlighting of the example code snippet at https://emberjs.com/api/ember-data/3.0/classes/DS.Store/methods/unloadAll?anchor=unloadRecord seems broken currently. The code looks alright, but the highlighting of the nested function seems to disagree for some reason.

**Comment by Turbo87** [comment]
@sivakumar-kailasam @toddjordan any idea **why** this is happening? not sure if this is indeed "good for new contributors" if it's not clear what the problem actually is 🤔

**Comment by toddjordan** [comment]
Yeah I don't know.  looks like a bug in our syntax highlighting library.  I don't know if tweaking the source will help.  We have talked about moving to prism in the past, but that's a bit beyond a "good for new contribs" issue.
